src/components/homePageCommonTasks/template.tsx

```diff
@@ -1,20 +1,28 @@
+import clsx from 'clsx'
 import { CommonTasksData } from './formatted-type'
+import styles from './template.module.css'
```

Comment:
Took a page from [Zach's playbook](https://github.com/department-of-veterans-affairs/next-build/blob/fc4c4a8f774fc808a4dcbb1013dc26e1dd314f46/src/components/campaignLandingPage/StoriesPanel.tsx#L4)